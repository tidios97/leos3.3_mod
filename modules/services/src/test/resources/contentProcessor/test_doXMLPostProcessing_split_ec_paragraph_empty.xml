<root xmlns:leos="urn:eu:europa:ec:leos" xmlns:xml="http://www.w3.org/XML/1998/namespace">
    <article leos:origin="ec" xml:id="art_1" leos:editable="false" leos:deletable="false" leos:softuser="demo">
        <num leos:origin="ec" leos:editable="false" xml:id="art_1__num">Article 1</num>
        <heading leos:origin="ec" xml:id="art_1__heading">Scope</heading>
        <paragraph leos:origin="ec" xml:id="art_1__para_1" leos:softuser="demo">
            <num leos:origin="ec" xml:id="art_1__para_1__num">1.</num>
            <subparagraph xml:id="akn_art_subpara_HaUTRs" leos:softaction="add" leos:softactionroot="true" leos:softuser="demo"
                          leos:softdate="2019-01-29T11:56:51.605+01:00">
                <content leos:origin="ec" xml:id="art_1__para_1__content">
                    <p leos:origin="ec" xml:id="art_1__para_1__content__p">Text...<inline leos:origin="ec" xml:id="art_1_0miExA" name="math-tex">\\(x = {-b \\pm \\sqrt{b^2-4ac} \\over 2a}\\)</inline>
                    </p>
                </content>
            </subparagraph>
            <subparagraph xml:id="akn_art_subpara_JwqJAz" leos:softaction="add" leos:softuser="demo" leos:softdate="2019-01-29T11:56:51.603+01:00"
                          leos:origin="cn">
                <content xml:id="art_1_dB5lte">
                    <p xml:id="art_1_WxnFDr">Subparagraph Text...</p>
                </content>
            </subparagraph>
        </paragraph>
    </article>
</root>