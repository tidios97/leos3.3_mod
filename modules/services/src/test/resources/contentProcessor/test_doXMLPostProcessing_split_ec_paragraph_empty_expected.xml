<root xmlns:leos="urn:eu:europa:ec:leos">
    <article leos:deletable="false" leos:editable="false" leos:origin="ec" leos:softuser="demo" xml:id="art_1">
        <num leos:editable="false" leos:origin="ec" xml:id="art_1__num">Article 1</num>
        <heading leos:origin="ec" xml:id="art_1__heading">Scope</heading>
        <paragraph leos:origin="ec" leos:softuser="demo" xml:id="art_1__para_1">
            <num leos:origin="ec" xml:id="art_1__para_1__num">1.</num>
            <subparagraph leos:origin="ec" leos:softaction="trans" leos:softactionroot="true"
                          leos:softdate="2022-06-17T09:33:27.945+02:00" leos:softuser="demo(DIGIT)"
                          xml:id="transformed_art_1__para_1">
                <content leos:origin="ec" xml:id="art_1__para_1__content">
                    <p leos:origin="ec" xml:id="art_1__para_1__content__p">Text...
                        <inline leos:origin="ec" name="math-tex" xml:id="art_1_0miExA">\\(x = {-b \\pm \\sqrt{b^2-4ac}
                            \\over 2a}\\)
                        </inline>
                    </p>
                </content>
            </subparagraph>
            <subparagraph leos:origin="cn" leos:softaction="add" leos:softdate="2019-01-29T11:56:51.603+01:00"
                          leos:softuser="demo" xml:id="akn_art_subpara_JwqJAz">
                <content xml:id="art_1_dB5lte">
                    <p xml:id="art_1_WxnFDr">Subparagraph Text...</p>
                </content>
            </subparagraph>
        </paragraph>
    </article>
</root>